### ISSUE TITLE
psp2-fixup: Pointer reference in .data to .rodata may be wrong

### ISSUE THREAD

**Issue by 173210** [psp2dev/psp2sdk#47]
It caused a problem with this code.
https://github.com/xerpi/FTPVita/blob/b1e2989472c72b22f471e887114d27adefdefdb5/ftp.c#L160

```
static const char *num_to_month[] = {
	"Jan", "Feb", "Mar", "Apr", "May", "Jun",
	"Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
};
```

**Comment by 173210** [comment]
Maybe fixed with commit abf8e7ad9ff759da9474a530898eb56053baf1cb.

**Comment by 173210** [comment]
Maybe fixed with commit 6ff73c675afcc8bd4856f1f79b1aef8e71b4e118.

**Comment by 173210** [comment]
Fixed with commit 014c70846c995e7215299c81dfd9c7969568beac.
